lms/static/js/groups/views/verified_track_settings_notification.js

```diff
@@ -0,0 +1,79 @@
+;(function (define) {
+    'use strict';
+    define(['jquery', 'underscore', 'backbone', 'gettext', 'edx-ui-toolkit/js/utils/string-utils',
+            'js/models/notification', 'js/views/notification'],
+        function ($, _, Backbone, gettext, StringUtils) {
+            /*global NotificationModel, NotificationView */
+
+            var VerifiedTrackSettingsNotificationView = Backbone.View.extend({
+
+                render: function () {
+                    // All rendering is done in validateSettings, which must be called with some additional information.
+                    return this;
+                },
+
+                validateSettings: function (isCohorted, cohortCollection, enableCohortsCheckbox) {
+                    if (this.model.get('enabled')) {
+                        var verifiedCohortName = this.model.get('verified_cohort_name');
+                        if (isCohorted) {
+                            var verifiedCohortExists = false;
+                            $.each(cohortCollection, function (_, cohort) {
+                                if (cohort.get('assignment_type') === 'manual' &&
+                                    cohort.get('name') === verifiedCohortName) {
+                                    verifiedCohortExists = true;
+                                    cohort.disableEditingName = true;
+                                }
```

Comment:
Nit: The indentation of this if-block's contents lining up with the last part of the condition threw me for a second on first read, could the body of the block get bumped inwards one more tab?